### DISCUSSION TITLE
Is it possible to change the isometric grid size?

### DISCUSSION THREAD

**Discussion by miltonism** [xournalpp/xournalpp#4527] [General] [answered]
Is there a way to change the isometric grid size to a smaller size?

I need finer points for my work, but the larger the paper size the more the software struggles to keep up and slows down a lot.

If I could make the grid size smaller, I could keep the paper size small.

**Comment by rolandlo** [comment, marked as answer]
Yes, it is possible by modifying `pagetemplates.ini` (parameter `r1`) , for example by appending the `config=r1= ...` line to the `predifinedIsoGraph` template (or by adding a new format):
```.ini
[predefinedIsoGraph]
name=Isometric Graph
name[de]=Isometrisches Raster
format=isograph
config=r1=10
```
Here 10 means 10pt = 10/72 inch.

**Reply by miltonism** [reply]
Excellent, just what I needed, thank you!
